nilearn/version.py

```diff
@@ -19,7 +19,10 @@
 # Dev branch marker is: 'X.Y.dev' or 'X.Y.devN' where N is an integer.
 # 'X.Y.dev0' is the canonical version of 'X.Y.dev'
 #
-__version__ = "0.10.1.dev"
+try:
+    from ._version import __version__
+except ImportError:
+    pass
```

Comment:
You may want to create a PEP440-compliant fallback like `"0+unknown"` so that you don't get a `NameError` or `AttributeError` when trying to use `__version__` if this fails for some reason.